models/articles.models.js

```diff
@@ -36,3 +36,23 @@ exports.fetchAllArticleComments = (article_id) => {
       return res.rows;
     });
 };
+
+exports.addCommentData = (addComment, article_id) => {
+  if (
+    addComment.author === undefined ||
+    addComment.body === undefined ||
+    article_id === undefined
```

Comment:
Lets allow psql to throw an error if it's going to, we only go to custom errors if psql doesn't. See what happens if you pass these invalid cases to the query.